src/test/librbd/test_librbd.cc

```diff
@@ -3419,7 +3423,7 @@ TEST_F(TestLibRBD, FlattenNoEmptyObjects)
   librbd::RBD rbd;
   std::string parent_name = get_temp_image_name();
   uint64_t size = 4 << 20;
-  int order = 12; // smallest object size is 4K
+  int order = 0;
```

Comment:
W/ the default order (4MB) and an image size of 4MB, would you expect any empty objects?